<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyImageWidgetClass</class>
 <widget class="QWidget" name="MyImageWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>657</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MyImageWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="labelInformation">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Source Code Pro&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStatus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Source Code Pro&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelImage">
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyImageWidget.qrc"/>
 </resources>
 <connections/>
</ui>
